src/components/DiagnosticCard/DiagnosticCard.tsx

```diff
@@ -4,12 +4,20 @@ import './DiagnosticCard.scss';
 
 const b = cn('ydb-diagnostic-card');
 
-interface DiagnosticCardProps {
+export interface DiagnosticCardProps {
     children?: React.ReactNode;
     className?: string;
     active?: boolean;
+    size?: 'm' | 'l' | 's';
+    interactive?: boolean;
 }
 
-export function DiagnosticCard({children, className, active}: DiagnosticCardProps) {
-    return <div className={b({active}, className)}>{children}</div>;
+export function DiagnosticCard({
+    children,
+    className,
+    active,
+    size = 'm',
+    interactive = true,
+}: DiagnosticCardProps) {
+    return <div className={b({active, size, interactive}, className)}>{children}</div>;
```

Comment:
Is this card supposed to be used somewhere else in the future
As for now for me it looks like a single-place wrapper for child components without clear self-worthness

Reply:
oh, I see where it is used, sorry